test/src/main/java/org/apache/accumulo/test/performance/RollWALPerformanceIT.java

```diff
@@ -59,6 +59,11 @@ protected void configure(MiniAccumuloConfigImpl cfg, Configuration hadoopCoreSit
     cfg.useMiniDFS(true);
   }
 
+  @Override
+  protected int defaultTimeoutSeconds() {
+    return 20 * 60;
```

Comment:
Wow! Do we seriously have a test we expect to take up to 20 minutes?!?!?! This seems awful.

Reply:
Ugh I know... I made a comment in #1482 that I think we should just drop this IT in 1.9